app/javascript/components/TiptapExtensions/UpsellCard.tsx

```diff
@@ -172,30 +173,29 @@ const UpsellCardNodeView = ({ node, selected, editor }: NodeViewProps) => {
         {isLoading ? (
           <div className="dummy h-32"></div>
         ) : product ? (
-          <article className="product-card horizontal">
-            <figure>
+          <ProductCard className="lg:flex-row">
+            <ProductCardFigure className="lg:aspect-auto lg:h-full lg:rounded-l lg:rounded-tr-none lg:border-r lg:border-b-0">
               <Thumbnail url={null} nativeType={product.native_type} />
-            </figure>
-
-            <section>
+            </ProductCardFigure>
+            <section className="flex flex-1 flex-col lg:gap-8 lg:px-6 lg:py-4">
               {isEditable ? (
                 <UpsellCardHeader product={product} variant={variant} />
               ) : (
                 <a href={getUpsellUrl(id ?? "", product.permalink)} className="stretched-link">
                   <UpsellCardHeader product={product} variant={variant} />
                 </a>
               )}
-              <footer className="text-base">
+              <ProductCardFooter className="*:lg:not-last:border-r-0">
                 {product.review_count > 0 ? (
-                  <div className="flex shrink-0 items-center gap-1">
+                  <div className="flex flex-[1_0_max-content] items-center gap-1 p-4 lg:p-0">
```

Comment:
Just wondering, do we want similar styles on the ratings item in `app/javascript/components/Product/Card.tsx`?